ingestfile

```diff
@@ -397,8 +409,13 @@ if [ "${PHASECHECKTEST}" = "Y" ] ; then
 fi
 
 #frame count test- will ask operator for what to do if there is a discrepancy between the container frame count and the video track frame count.
-_get_frame_count "${INPUT}"
-
+if ! -z _get_frame_count "${INPUT}" && [ "${SILENT_RUNNING}" == "Y" ] ; then
```

Comment:
This section seems to move the _get_frame_count section into ingestfile, maybe better to add a `if not silent running` part to the function to prevent it from asking for feedback